### PR TITLE
Adding doc about scripts as OneDrive files

### PR DESCRIPTION
**Pull request by AlexJerabek**
This aims to clarify how script files are stored on the user's OneDrive.

### CODE REVIEW

File: docs/overview/script-storage.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+# Office Scripts file storage and ownership
+
+Office Scripts are stored as **.osts** files in your OneDrive. The permissions of a script inside a workbook are limited to the permissions of that script's owner. The script **.osts** files carry whatever shared access and permissions that user's OneDrive allows.
+
+## File storage
+
+You Office Scripts are stored in your OneDrive. The **.osts** files are found in the **/Documents/Office Scripts/** folder. Any edits made these **.osts** files, such as renaming or deleting files, will be reflected in the Code Editor and Script Gallery.
+
+Scripts that are shared with one of your workbooks remain in the cloud and are not locally copied. The **Make a Copy** button of the Code Editor saves a separate, local copy of the script.Changes to the copy do not affect the original script.
```
Comment (peronald) [inline docs/overview/script-storage.md:16]: Space needed before "Changes to the copy..

Comment (peronald) [reply docs/overview/script-storage.md:16]: We should say Make a copy button of the Code Editor saves a separate copy of the script stored in the user's OneDrive (it isn't local).

File: docs/overview/script-storage.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: 'Office Scripts file storage and ownership'
+description: 'Information about how Office Scripts are stored in OneDrive and transferred between owners.'
+ms.date: 11/13/2020
+localization_priority: Normal
+---
+
+# Office Scripts file storage and ownership
+
+Office Scripts are stored as **.osts** files in your OneDrive. The permissions of a script inside a workbook are limited to the permissions of that script's owner. The script **.osts** files carry whatever shared access and permissions that user's OneDrive allows.
```
Comment (peronald) [inline docs/overview/script-storage.md:10]: What are we trying to say by "The permissions of a script inside a workbook are limited to the permissions of that script's owner?

**Comment by AlexJerabek** [comment]
Thanks for the feedback @peronald. I've updated the PR accordingly. Is there anything else we should include on this topic? If not, I'll promote this from draft status and resolve the associated issues.

File: docs/overview/excel.md [hunk 3]
```diff
@@ -66,7 +66,7 @@ Office Scripts can be shared with other users of an Excel workbook. When you sha
 More details about sharing and unsharing scripts can be in the article [Sharing Office Scripts in Excel for the Web](https://support.microsoft.com/office/sharing-office-scripts-in-excel-for-the-web-226eddbc-3a44-4540-acfe-fccda3d1122b).
 
 > [!NOTE]
-> Since Office Scripts are stored on a user's OneDrive, they follow the same retention and deletion policies. To learn how to handle scripts that were created and shared by a user being removed from your organization, see [OneDrive retention and deletion](/onedrive/retention-and-deletion).
+> Learn more about scripts are stored in your OneDrive in [Office Scripts file storage and ownership](script-storage.md).
```
Comment (davidchesnut) [inline docs/overview/excel.md:69]:
```suggestion
> Learn more about how scripts are stored in your OneDrive in [Office Scripts file storage and ownership](script-storage.md).
```

File: docs/overview/script-storage.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+---
+title: 'Office Scripts file storage and ownership'
+description: 'Information about how Office Scripts are stored in OneDrive and transferred between owners.'
```
Comment (davidchesnut) [inline docs/overview/script-storage.md:3]:
```suggestion
description: 'Information about how Office Scripts are stored in Microsoft OneDrive and transferred between owners.'
```

File: docs/overview/script-storage.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+ms.date: 11/13/2020
+localization_priority: Normal
+---
+
+# Office Scripts file storage and ownership
+
+Office Scripts are stored as **.osts** files in your OneDrive. This allows your scripts to exist outside any particular workbook. Your OneDrive settings control the shared access and permissions for these script **.osts** files; independent of any Excel settings.
```
Comment (davidchesnut) [inline docs/overview/script-storage.md:10]:
```suggestion
Office Scripts are stored as **.osts** files in your Microsoft OneDrive. This allows your scripts to exist outside any particular workbook. Your OneDrive settings control the shared access and permissions for all script **.osts** files; independent of any Excel settings.
```

File: docs/overview/script-storage.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+## File storage
+
+You Office Scripts are stored in your OneDrive. The **.osts** files are found in the **/Documents/Office Scripts/** folder. Any edits made these **.osts** files, such as renaming or deleting files, will be reflected in the Code Editor and Script Gallery.
```
Comment (davidchesnut) [inline docs/overview/script-storage.md:14]:
```suggestion
You Office Scripts are stored in your OneDrive. The **.osts** files are found in the **/Documents/Office Scripts/** folder. Any edits made to these **.osts** files, such as renaming or deleting files, will be reflected in the Code Editor and Script Gallery.
```

File: docs/overview/script-storage.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+Scripts that are shared with one of your workbooks remain in the cloud and are not copied to any of your local or OneDrive folders. The **Make a Copy** button of the Code Editor saves a separate copy of the script in your OneDrive. Changes to the copy don't affect the original script.
```
Comment (davidchesnut) [inline docs/overview/script-storage.md:16]: It's a little ambiguous to me what the difference is between a script "in the cloud" and "in your OneDrive". Is there another cloud file system option other than OneDrive?

File: docs/overview/script-storage.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+### Script Folders
```
Comment (davidchesnut) [inline docs/overview/script-storage.md:18]:
```suggestion
### Script folders
```

File: docs/overview/script-storage.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+
+Adding folders to your OneDrive helps keep your scripts organized. Any folders under **/Documents/Office Scripts/** are displayed under the **My Scripts** section of the Code Editor. Please note that these folders cannot be created and deleted through the Code Editor. Likewise, scripts cannot be placed in to or out of folders through the Code Editor.
```
Comment (davidchesnut) [inline docs/overview/script-storage.md:20]:
```suggestion
Adding folders to your OneDrive helps keep your scripts organized. Any folders under **/Documents/Office Scripts/** are displayed under the **My Scripts** section of the Code Editor. Please note that these folders cannot be created or deleted by using the Code Editor. Likewise, scripts cannot be placed in folders, or moved across folders by using the Code Editor.
```
The "placed out of folders" was ambiguous to me. Is this change what you meant?

File: docs/overview/script-storage.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+
+![Some scripts contained in folders, as displayed in the Code Editor task pane](../images/script-folders.png)
+
+## File ownership and retention
+
+Office Scripts are stored on a user's OneDrive, they follow the same retention and deletion policies. To learn how to handle scripts that were created and shared by a user being removed from your organization, see [OneDrive retention and deletion](/onedrive/retention-and-deletion).
```
Comment (davidchesnut) [inline docs/overview/script-storage.md:26]:
```suggestion
Office Scripts are stored in a user's OneDrive. They follow the same retention and deletion policies. To learn how to handle scripts that were created and shared by a user being removed from your organization, see [OneDrive retention and deletion](/onedrive/retention-and-deletion).
```
Can you clarify which retention and deletion policies are being followed (second sentence)? "same" as which ones?

**Review by davidchesnut — CHANGES_REQUESTED** [review]
Looking good. I have some editorial feedback. Thanks!

File: docs/overview/script-storage.md (hunk 9 above)
Comment (AlexJerabek) [reply docs/overview/script-storage.md]: More or less. They also can't be taken out of folders, but I think that's implied by your version.

File: docs/overview/script-storage.md (hunk 7 above)
Comment (AlexJerabek) [reply docs/overview/script-storage.md]: No, it's all OneDrive, but it centers around which OneDrive is being used. How about:
"Scripts that are shared with one of your workbooks remain in the script creator's OneDrive. They are not copied to any of your local or OneDrive folders when you run the shared script in Excel."

**Comment by AlexJerabek** [comment]
Thanks @davidchesnut. I've applied your feedback and believe to have addressed your remaining questions.

**Review by davidchesnut — APPROVED** [review]
Looks great!

**Pull request merged**